web/public/js/directives/event/cd-create-event/template.dust

```diff
@@ -235,6 +235,11 @@
           <label for="ticketApproval">{@i18n key="Turn on ticket approval"/}</label>
           <i uib-tooltip="{@i18n key="Enabling this will require you or another admin with Ticketing Admin permissions to approve applications to attend this event."/}" tooltip-placement="top" class="fa fa-info-circle event-tooltip"></i>
         </div>
+        <div class="checkbox checkbox-block">
```

Comment:
@rosalanghammer Are you ok with "Send me email notifications" for the event tickbox to receive.. notifications?